gossip/src/cluster_info.rs

```diff
@@ -1470,16 +1443,23 @@ impl ClusterInfo {
             .collect()
     }
 
-    fn insert_self(&self) {
-        let node = self.my_contact_info();
+    fn refresh_my_gossip_contact_info(&self) {
+        let keypair: Arc<Keypair> = self.keypair().clone();
```

Comment:
Nice, reduced down to one call to `Self::keypair()` instead of doing it per-item